### DISCUSSION TITLE
심화프로젝트 4주차 팀회고

### DISCUSSION THREAD

**Discussion by sulyoungoh** [codeit-bootcamp-frontend/3-Weekly-Mission#348] [회고]
벌써 심화 프로젝트 4주차 팀 회고입니다 여러분🥹
팀 회고 후([회고 방식 상세보기](https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/discussions/346#discussion-6517270)), 이전처럼 댓글 남겨주세요🙏

마지막 프로젝트인만큼 꼼꼼하게 기록하여 나와 팀원들의 모습을 남겨보세요 ☺️
남은 4주차도 파이팅💪

**Comment by jinahp** [comment]
# 6팀

## K (Keep)
- 팀원들이 너무 좋다
- 모두가 회의할 때 의견을 정말 잘 제시한다.
- 불성실한 사람이 단 1명도 없다.
- 코딩이야기를 할 때 화면공유가 바로바로 이루어져 좋다 
- 다들 의견도 적극적으로 나누고 마지막을 좋은 팀원분들과 보낼 수 있어 너무나도 감사하게 생각한다!

## P (Problem)
- 이제 수료하면 못 본다니 아쉽다
- ㄴ또르륵.....
- 이번주는 문제가 없다 다들 정말 열심히 잘 하고 있다
- 일의 재분배가 필요해보인다

## T (Try)
- 함께하는 이 순간을 소중하게 보내자
- 지금 속도로 킵고잉!! 💯💯
- 다음주 모든 필수적인 ui & 기능에 대한 진행사항을 하나하나 점검 해보자

**Comment by yunsunglee2** [comment]
# 7팀

## K (Keep)
- 팀원들과의 각자 진행상황 공유를 많이 해서 전체적으로 프로젝트의 남은 부분을 알 수 있다. 
- 기한까지 최선을 다해서 제출하기
- 꼼꼼하게 작업하려고 노력

## P (Problem)
- 마감일에 맞춰 몰아서 작업을 한거 같다. 시간계획을 세분화해서 효율적으로 작업하고 싶다
- api 생성하기 전에 백엔드분들과 소통이 적었던거 같다. api가 나오고 수정사항이 많이 생김
- 디자인과 백엔드 설계를 기다리는 시간에 긴장이 조금 풀렸던거 같다

## T (Try)
- 컴포넌트 설계를 좀 더 잘했거나 api 응답 데이터에 대해서 내려주기전에 얘기를 많이 했으면 시간이 많이 단축 되었을 것 같음
- 시간이 많이 남았다고 생각했지만 일정이 빠듯해 미루지않고 작업
- 백엔드에 요청하고 싶은게 있다면 너무 오래 시간을 끌지 말고 상의하기! 
- 모는게 있으면 멘토님께 더 적극적으로 여쭤보기
- 부가사항에 대해서 고민하기

**Comment by Dev-Duke-Seo** [comment]
# 4팀
# K (Keep)
- 미팅시간 외에도 기획이나 파트간 소통이 늘어난 모습이 좋습니다.
- 게더 타운 활용해서 서로 소통하는 모습이 좋습니다.
- 요구사항이 바로 반영이 되어 작업할 때 편합니다.
- 개발이 순차적으로 잘 진행되어 가는것 같아 보기 좋습니다.
# P (Problem)
- 다른 파트의 작업 진행 상황을 정확히 알기 어려운 것 같습니다.
- 혼자서만 문제를 해결하려고 했던 것 같습니다.
- 개발하는데 드는 실제 소요 시간이 예상했던 기간보다 길어지는 것 같습니다.
- 학습이 필요한 부분이 많은 것 같습니다.
- 코드 리뷰하는데 집중이 잘 안되고 눈에 잘 안들어옵니다
- 일정을 계획하고 공유하는게 원활하지않습니다.
# T (Try)
- 데일리뿐만 아니라 파트별 백로그에서도 진행상황을 잘 업데이트하면 좋을 것 같습니다.
- 디스코드를 활용해서 팀원들과 함께 문제를 해결하도록 노력 해야겠습니다.
- 코드 양이 늘어나지 않게 pr을 자주 올리겠습니다.
- 코드리뷰를 미루지 말고 공부한다는 느낌으로 해보겠습니다.
- 계획은 다른 팀원도 잘 알  수 있도록 가시성있게 작성하고, 변경이나 문제가 생기면 바로 공유가 필요합니다.

**Comment by ohddang** [comment]
# 3팀

## K 
- 새로운 기술(또는 역량) 적용하기
- 각자 맡은 부분 잘 구현함
- Git issue 작성 후 작업

## P 
- 본인만 알고 있는 내용 공유 필요
- 새로고침 시 에러가 보임
- 팀원들이 작성한 코드 아직 완전히 이해하지 못함
- 백엔드, 인프라 부분 조언받기 어려움

## T
- 실제로 AWS에 올려서 Test후 버그 및 기능 수정
- 모든 서버 동작시킨 후 작업
- 노션에 지식공유 및 PR 꼼꼼히 보기
- 테스트 코드 더 짜기

**Comment by eundin** [comment]
# 2팀

## K
- 매일 각자 진행 상황 공유하기
- 회의때 적극적으로 의견내는것
- 기한이 얼마남지않았는데 최선을 다하기
- 빠진것 없나 꼼꼼하게 코드짜기
- 발전하려고 노력함

## P
- 속도가 더디고 각자 진행 상황이 서로 공유가 잘 안되고 있는 것 같다.
- 질문이 생기면 팀미팅까지 모아두는게 아닌 실시간으로 물어보면 시간절약이 될 것 같습니다.
- 기한에 맞춰서 하려다 보니 오히려 더 늘어지는것 같다.

## T
- 해결이 안되는 문제가 있을 때 혼자 고민하는 시간을 오래 갖지 말고 공유하고 멘토님께 질문도 많이 했으면 좋겠다.
- 질문 쌓아두지 않기
- 모르는 거나 궁금한게 생기면 팀원들끼리 공유하기
- 모르는게 있으면 멘토님께 적극적으로 물어보자

**Comment by Song-Sang** [comment]
# 1팀
# K (Keep)
- 각자 해야 할 일을 잘 찾아서 개발하고 있어서 회의 시간이 줄어들어서 좋다.
- 새로운 기능적인 부분이나 디자인적인 부분을 수정 할 때나 잘 생각해보고 본인의 의견을 내주셔서 피드백이 빠르다.
- 개발 속도가 다들 생각보다 빨라서 추가적인 기능들을 이것저것 구현하고 계셔서 좋다.

# P (Problem)
- 어느 정도 기능적인 부분들이 완료되어서 그런지, PR제출과, 리뷰 부분이 초반처럼 원활하지 않은 것 같다.
- 기능은 이상 없지만 불필요한 코드들이나, 지저분한 코드들이 자주 보이는 것 같다.
- 각자 자기 페이지 위주로 개발을 하게 되다 보니, 다른 페이지의 코드 상태를 아는 것이 PR리뷰 만으로는 조금 부족했던 듯 싶다.

# T (Try)
- PR 리뷰에 대한 부분은 각자 조금 더 의식적으로 신경 쓰기.
- 남은 기간 본인 파트가 어느 정도 마무리 되었다고 생각하면 리팩토링 진행하기.
- 의아한 부분이 생기면 서로 질문 자주 하기.
- 리팩토링 진행하면서 다른 페이지 코드들도 자세히 살펴보는 시간 가지기.

**Comment by DanteSnow** [comment]
5팀

# **👍🏻**Keep

---

- 오류 공유하면 다 같이 모여서 해결하려고 하는 것이 너무 좋다. 너무 큰 도움이 된다.
- 모여서 같이 하면서 다들 열심히 하시는 모습에 동기부여가 되는 것 같습니다.
- 늦은 시간까지 함께 작업하며 팀 프로젝트 활동에 열정을 불어넣는 팀원들 덕분에 열심히 하게 되는 것 같습니다.
- 친구, 가족같은 밝은 분위기가 형성되어 함께 있는 것이 즐겁습니다.

## **🤔**Problem

---

- 맘이 너무 급하다
- 소통을 위한 접점이 되는 시간이 부족한 것 같습니다.
- 백엔드와의 소통이 부족한 것 같습니다. 작업 일정들과 현황들이 파악이 되지 않아서 마음이 조급해집니다.
- 원래 정한 작업 일정보다 지연이 많이 되고 있어서 불안합니다. 대책이 필요할 것 같습니다.

## **🔥**Try

---

- 차근 차근 하나씩 쳐내야지
- 모르는 게 있으면 물어보고 공유하는 습관을 들이면 좋을 것 같습니다. 문제 사항, 시도한 방법을 정리해서 질문하면 문제사항을 이해하는 데 도움이 될 것입니다.
- 회의록 또는 칸반보드를 통하여 작업일정을 공유하여 작업 소통을 할 수 있으면 좋을 것 같습니다.
- 구체적인 작업 일정을 공유해 주셔서 다른 분들이 참고할 수 있게 해주셨으면 좋겠습니다.
